TYPO3.Neos/Classes/TYPO3/Neos/Eel/FlowQueryOperations/SortOperation.php

```diff
@@ -0,0 +1,117 @@
+<?php
+namespace TYPO3\Neos\Eel\FlowQueryOperations;
+
+/*
+ * This file is part of the TYPO3.Neos package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use TYPO3\Eel\FlowQuery\FlowQuery;
+use TYPO3\Eel\FlowQuery\Operations\AbstractOperation;
+use TYPO3\Flow\Annotations as Flow;
+use TYPO3\TYPO3CR\Domain\Model\Node;
+use TYPO3\TYPO3CR\Domain\Model\NodeInterface;
+
+/**
+ * "sort" operation working on TYPO3CR nodes.
+ * Sorts nodes by specified node properties.
+ */
+class SortOperation extends AbstractOperation
+{
+
+    /**
+     * {@inheritdoc}
+     *
+     * @var string
+     */
+    protected static $shortName = 'sort';
+
+    /**
+     * {@inheritdoc}
+     *
+     * @var integer
+     */
+    protected static $priority = 1;
```

Comment:
You're right, sorry. It's just that almost all of those we provide in Neos use `100`, but looking at it it's actually a bit of a mess. Some have `0` (hmm?) others `1` and the rest `100`. It's better to just leave it out as you did if there isn't a default operation with the same name. We should probably streamline those priorities.